chaos.py

```diff
@@ -69,12 +74,17 @@ def main():
 
     api = gh.API(settings.GITHUB_USER, settings.GITHUB_SECRET)
 
+    # Api Twitter
+    api_twitter = ta.API_TWITTER(settings.TWITTER_API_KEYS_FILE)
+
     log.info("checking if I crashed before...")
+    ta.Twitter.PostTwitter("checking if I crashed before...", api_twitter.GetApi())
```

Comment:
I think like @rudehn. Anything posted to twitter should be important. A post in twitter isn't a cost or some extra.